web/style.css

```
@@ -1230,3 +1230,65 @@ input[type="search"]::placeholder { font-size: .82em; }
+/* 本文の中のリンク。いまは書体の名前(LINE Seed JP)だけが使う。下の
+   .log-commit と違い、地の文に混ざるので下線で示す。 */
+.log-link { color: var(--sign-blue); text-decoration-thickness: 1px; }
+.log-link:visited { color: var(--sign-blue); }
+.log-link:hover { color: var(--ink); }
+
+/* コミットの短い名前。共有ダイアログの URL 欄と同じ等幅にする。1 字ずつ突き
+   合わせる字だからである。 */
+.log-commit {
+  font-family: ui-monospace, Consolas, monospace;
+  font-size: .72em; color: var(--sign-blue); text-decoration: none;
+  border-bottom: 1px solid currentColor; margin-left: .4em;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)